packages/sdk/src/Vouchers.ts

```diff
@@ -23,55 +25,55 @@ export class Vouchers {
 	/**
 	 * @see https://docs.voucherify.io/reference?utm_source=github&utm_medium=sdk&utm_campaign=acq#create-voucher
 	 */
-	public create(voucher: $FixMe) {
-		return this.client.post(`/vouchers/${encode(voucher.code)}`, voucher)
+	public create(voucher: T.VouchersCreate) {
+		return this.client.post<T.VouchersCreateResponse>(`/vouchers/${encode(voucher.code)}`, voucher)
 	}
 	/**
 	 * @see https://docs.voucherify.io/reference?utm_source=github&utm_medium=sdk&utm_campaign=acq#vouchers-get
 	 */
 	public get(code: string) {
-		return this.client.get(`/vouchers/${encode(code)}`)
+		return this.client.get<T.VouchersGetResponse>(`/vouchers/${encode(code)}`)
 	}
 	/**
 	 * @see https://docs.voucherify.io/reference?utm_source=github&utm_medium=sdk&utm_campaign=acq#update-voucher
 	 */
-	public update(voucher: $FixMe) {
-		return this.client.put(`/vouchers/${encode(voucher.code)}`, voucher)
+	public update(voucher: T.VouchersUpdate) {
+		return this.client.put<T.VouchersUpdateResponse>(`/vouchers/${encode(voucher.code)}`, voucher)
 	}
 	/**
 	 * @see https://docs.voucherify.io/reference?utm_source=github&utm_medium=sdk&utm_campaign=acq#delete-voucher
 	 */
-	public delete(code: string, params?: { force: boolean }) {
-		return this.client.delete(`/vouchers/${encode(code)}`, { force: !!params?.force })
+	public delete(code: string, params: T.VouchersDeleteParams = {}) {
+		return this.client.delete(`/vouchers/${encode(code)}`, params)
 	}
 	/**
 	 * @see https://docs.voucherify.io/reference?utm_source=github&utm_medium=sdk&utm_campaign=acq#list-vouchers
 	 */
-	public list(params?: $FixMe) {
-		return this.client.get('/vouchers', params)
+	public list(params: T.VouchersListParams = {}) {
```

Comment:
Yep, the seralization was off. 

I had to install Qs library for this to work. I've added paramsSerialize to GET method of RequestController, so it will work with other list methods.

Tommorow morning I will check it with other list methods + I will check if those new methods (bulkUpdate, bulkUpdateMetadata) are working correctly.